### DISCUSSION TITLE
Anonymous tag

### DISCUSSION THREAD

**Discussion by eerison** [markdoc/markdoc#358] [Q&A] [answered]
Could I define an empty tag and it wrap the content 

For example 

```
{% .myclass %}
ABC
{% /.myclass %}
```

And it parse to 

```html
<div class="myclass">ABC</div>
```

**Comment by rpaul-stripe** [comment, marked as answer]
No, you can't use a class annotation as an opening and closing element. You can, however, put a class annotation directly inside of a block to apply it to the block itself.

For example:

```
ABC {% .myclass %}
```

Is effectively:

```
<p class="myclass">ABC</p>
```

If you want a div wrapper you would have to define a custom named tag:

```
{% example .myclass %}
ABC
{% /example %}
```

**Reply by eerison** [reply]
Thanks for clarify 🙂
